js/ui/file_manager/ui.file_manager.toolbar.js

```diff
@@ -11,6 +11,55 @@ const FILE_MANAGER_GENERAL_TOOLBAR_CLASS = "dx-filemanager-general-toolbar";
 const FILE_MANAGER_FILE_TOOLBAR_CLASS = "dx-filemanager-file-toolbar";
 const FILE_MANAGER_TOOLBAR_SEPARATOR_ITEM_CLASS = FILE_MANAGER_TOOLBAR_CLASS + "-separator-item";
 
+const DEFAULT_TOOLBAR_FILE_ITEMS = [
+    {
+        commandName: "download",
```

Comment:
Use a localization constant!

Reply:
This code already committed in the repository. The changes is result of small refactoring. I think, that is better way create new branch and find all text constant. If I change this code lines I don't solve localization issue completely. What do you think about it?

Reply:
ok